newdle/client/src/components/summary/SummaryPage.js

```diff
@@ -48,6 +48,8 @@ export default function SummaryPage() {
     client.sendResultEmails
   );
   const [_setFinalDate, submitting] = client.useBackendLazy(client.setFinalDate);
+  const [_deleteNewdle, deleting] = client.useBackendLazy(client.deleteNewdle);
```

Comment:
Looks like something went wrong during your rebase - this was moved elsewhere.

Reply:
Removed.